bonsai/sdk/src/alpha.rs

```diff
@@ -129,8 +129,8 @@ pub mod responses {
         pub state: Option<String>,
```

Comment:
Can we make this an enum?

Reply:
Currently it cannot, it has some dynamically generated components.